SocietalConstructionTool/Compiler/SctTranslator.cs

```diff
@@ -302,7 +304,7 @@ private static BinaryExpressionSyntax ConvertToBooleanCondition(ExpressionSyntax
         public override void ExitLiteralExpression([NotNull] SctParser.LiteralExpressionContext context)
         {
             // if value is int, double will print without decimal point
-            var value = double.Parse(context.LIT().GetText());
+            var value = double.Parse(context.LIT().GetText(), CultureInfo.InvariantCulture);
```

Comment:
Also implemented in #68, will make a conflict

Reply:
cope